manifests/s/SanJin/thief/4.0.0/SanJin.thief.locale.zh-CN.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SanJin.thief
+PackageVersion: 4.0.0
+PackageLocale: zh-CN
+Publisher: SanJin
+PublisherUrl: https://sanjin.dev/
+PublisherSupportUrl: https://sanjin.dev/
+PrivacyUrl: https://sanjin.dev/
+Author: SanJin
+PackageName: Thief
+PackageUrl: https://github.com/cteamx/Thief
+License: GPL-3.0 license
+LicenseUrl: https://github.com/cteamx/Thief/blob/master/LICENSE
+Copyright: Copyright © 2020 SanJin
```

Comment:
Shouldn't the copyright be translated to match the localization?

Reply:
Yes, You can write-in it

Reply:
Thanks but I know too few Chinese characters to be able to translate effectively, and am concerned an internet translation might accidentally give an incorrect meaning.